Book (#262)
epiphany-platform/CommonDataLayer: docs/README.md

# Common Data Layer

> The Common Data Layer (CDL) is a data storage service. It is designed with performance, versatility,
> scalability, and ease-of-modification as key tenets of its design, among others.

Reviewer: In the future, please use this account (`smores56`) if you want me to get a notification.

I agree, generally it is good practice to avoid repetition with such a high degree of locality unless trying to make a point. In this case, the original sentence derives from the original documentation in Confluence written by Jakub. We can remove the "key tenets of its design" section, rendering something like:

```suggestion
The Common Data Layer (CDL) is a data storage service. Its primary goals are performance, versatility, scalability, and ease-of-modification.
```
